javascript_nodejs/4.simple-prompt-bot/dialogs/mainDialog/index.js

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This sample needs to be simplified. Here's the [spec](https://github.com/Microsoft/BotBuilder-Samples/blob/v4/readme.md) - 
```markdown
Demonstrates prompt pattern by prompting user for a property. Introduces user state .vs. conversation state. Ask for name and prints back those information. 
```
So I'd recommend that we pare back this sample and simplify it to just do a name prompt.

Reply:
Doh! I hadn't seen this document.  Thanks.